src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Config/SwaggerConfig.cs

```diff
@@ -0,0 +1,92 @@
+using System.Text.Json;
+using Asp.Versioning.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Config;
+
+public class SwaggerConfig(IApiVersionDescriptionProvider provider) : IConfigureOptions<SwaggerGenOptions>
```

Comment:
how does this provider get injected in?

Reply:
This is registered in the DI container as:

```cs
builder.Services.AddTransient<IConfigureOptions<SwaggerGenOptions>, SwaggerConfig>();
```

So it automatically wires it in for us :smile: